src/components/sidebar/cart/Cart.js

```diff
@@ -46,7 +53,23 @@ function Cart({ classes }) {
           {' '}
           {label}
         </Typography>
-        {!!uniqueSections.length && <Button onClick={handleClick}> Remove All </Button>}
+        {!!uniqueSections.length && <Button onClick={handleClickOpen}> Remove All </Button>}
+        <Dialog
+          open={open}
+          onClose={handleClose}
+          aria-labelledby="alert-dialog-title"
```

Comment:
Need to replace the aria attributes with an actual title and description

Reply:
Can I just take them out? I think I took others out but forgot that one when copying from the material-UI website

Reply:
You technically could? Aria labelling is for accessibility (eg when people who are less sighted use screen readers), but I'm not sure how big of a focus we have on that @Joonpark13 I feel like we should do it when we can, I don't remember if we keep it up with other parts of the codebase